From c1ca5ec4a5c6a0e4acbdcc6ff4e4fa2109c1ec24 Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Wed, 30 Jul 2025 14:13:07 -0700
Subject: [PATCH] meson: use curses for the non wide version

The curses dependency in meson in special in that it uses a combination
of pkg-config, config-tool, and various system lookups to find it.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/meson.build
+++ b/meson.build
@@ -289,7 +289,7 @@ if lib_ncursesw.found()
   lib_ncurses = disabler()
 else
   lib_ncurses = dependency(
-    'ncurses',
+    'curses',
     disabler : true,
     required : get_option('ncurses'))
   headers += ['ncurses.h',
